app/models/school.rb

```diff
@@ -186,6 +186,7 @@ class ProcessDataError < StandardError; end
   scope :inactive,            -> { where(active: false) }
   scope :deleted,             -> { inactive.where.not(removal_date: nil)}
   scope :archived,            -> { inactive.where(removal_date: nil)}
+  scope :active_and_archived, -> { where(removal_date: nil)}
```

Comment:
Just checking - is ok not not check the active flag here - and rely on the removal date being nil as the primary indicator of being active or archived?

Reply:
agree, might be clearer to include active + archived more explicitly

Reply:
:+1: